### PR TITLE
REST-API: Update expected user balance of last test

### PR DESCRIPTION
**Pull request by davidelettieri**
User balance of test "lender owes borrower same as new loan" is not formatted as the previous cases. For example "lender has negative balance" has an expected user balance of zero formatted as "0.0" as all other values while the last test has zero formatted as "0".

### CODE REVIEW

File: exercises/rest-api/canonical-data.json [hunk 1]
```diff
@@ -396,18 +396,18 @@
...
                             }
                         ]
                     }
                 }
             ]
         }
     ]
-}
\ No newline at end of file
+}
```
Comment (rpottsoh) [inline exercises/rest-api/canonical-data.json]: A small nit pick, but this change is not related to the other change you made earlier in the file.

**Comment by rpottsoh** [comment]
Oh, and make sure you clear up the issue that Travis CI is having.

File: exercises/rest-api/canonical-data.json (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/rest-api/canonical-data.json]: I've created an issue for this, as it is a common issue with PR's created using the GitHub interface: https://github.com/exercism/problem-specifications/issues/1443

**Comment by ErikSchierboom** [comment]
> Oh, @davidelettieri you should look into and see what the issue is that Travis CI is having.

The issue is that the version number should be incremented.

**Comment by ErikSchierboom** [comment]
Looking great!
